checkov/terraform/checks/resource/gcp/GoogleComputeSerialPorts.py

```diff
@@ -6,7 +6,7 @@ class GoogleComputeSerialPorts(BaseResourceNegativeValueCheck):
     def __init__(self):
         name = "Ensure 'Enable connecting to serial ports' is not enabled for VM Instance"
         id = "CKV_GCP_35"
-        supported_resources = ['google_compute_instance']
+        supported_resources = ['google_compute_instance', 'google_compute_instance_template']
```

Comment:
```suggestion
        supported_resources = ['google_compute_instance', 'google_compute_instance_template', 'google_compute_instance_from_template']
```
also negative check

Reply:
Makes sense

Reply:
I suppose, if a `from_template` instance passes a negative check condition, that doesn't mean that the underlying `template` will also pass the check. But that won't be solved until we have a graph check for it 🤷🏽‍♂️

Reply:
that's also true, only a graph check could help here